core/markdown_generator.py

```
@@ -3,16 +3,18 @@
 """
 
 from datetime import datetime
-from typing import Optional, Dict, Any
+from typing import Optional, Dict, Any, List
```

Ruff (0.8.0)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)